### PR TITLE
fix overview descriptions and canvas button

### PR DESCRIPTION
**Pull request by philipp-meier**
Thank you for this template!

I noticed that the section descriptions in the English "template overview" were mixed up and adjusted them to match the German version.

### CODE REVIEW

**Comment by philipp-meier** [comment]
I also added some fixes that were reported by my IDEs spell-checker since I already had the code open and tried to fix the "canvas button":

**Before**:
image

**After**:
image

File: assets/css/button.css
Comment (gernotstarke) [inline assets/css/button.css:1]: I like to leave it with 90% viewport

**Review by gernotstarke — APPROVED** [review]

**Comment by gernotstarke** [comment]
thanx for the effort!

**Comment by philipp-meier** [comment]
I'm glad if it helps.
Please let me know if something needs to be changed or rolled back.

**Pull request merged**